recipes/z3/all/conanfile.py

```diff
@@ -60,7 +72,8 @@ def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
-        self._cmake.definitions["Z3_USE_LIB_GMP"] = True
+        self._cmake.definitions["Z3_USE_LIB_GMP"] = self.options.multiprecision != "internal"
+        self._cmake.definitions["Z3_USE_LIB_MPIR"] = self.options.multiprecision == "mpir"
```

Comment:
I was not able to find this option with upstream https://github.com/Z3Prover/z3/search?q=Z3_USE_LIB_MPIR

🤔 where does it come from?

Reply:
The option is added by patches to switch between gmp/mpir.

Reply:
Ahh, this PR is correcting the recipe to better match upstream! It's supposed to be gmp

Thank you